src/components/Navigation/NavigationItem/_navigationItem.scss

```diff
@@ -69,6 +69,12 @@
     margin-bottom: 0;
   }
 
+  a {
+    @include isHeaderMobile {
+      display: block;
```

Comment:
This take care of mobile but you still have the pointer on desktop when its not on the link. If we take the cursor pointer off the `li` it will just turn to pointer when on anchor.